src/components/Checkbox.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { StyleSheet, Text, Pressable, View } from 'react-native'
+
+import { colors } from '../config/themes/appThemes'
+
+interface CheckboxProps {
+  onChange: (value: string) => void
+  value: string
+}
+
+export const Checkbox: React.FC<CheckboxProps> = ({ onChange, value }) => {
```

Comment:
No aplicar destructuración de esta forma. Esto con el fin de que todos los componentes tenga una estructura similar.

 ```suggestion
export const Checkbox: React.FC<CheckboxProps> = (props) => {
```